docs/ruleset_functions/aggregate_min_OA_schedule_across_zones.md

```diff
@@ -0,0 +1,26 @@
+# aggregate_min_OA_schedule_across_zones
+
+**Description:** Each HVAC system can serve multiple zones which can have multiple terminal units serving each zone. In order to obtain an hourly HVAC level OA CFM schedule the OA CFM provided by each zone needs to be aggregated for each hour of the year.  This function receives an RMR (B, U, or P) and a list of zone OA min cfm schedules (generated by the get_min_OA_CFM_sch_zone function) and loops through each zone to create an aggregated 8760 for OA CFM for the HVAC system. 
+
+**Inputs:**  
+- **U,P,or B-RMI**: The RMI in which the OA CFM schedule will be determined based on the list of zone OA CFM schedules sent to this function.  
+- **zone_OA_CFM_list_of_schedules**: A list which contains each zone's 8760 OA CFM schedule as a list value.  
+
+**Returns:**  
+- **aggregate_min_OA_schedule_across_zones**: TThis function receives an RMR (B, U, or P) and a list of zone OA min cfm schedules (generated by the get_min_OA_CFM_sch_zone function) and loops through each zone to create an aggregated 8760 for OA CFM for the HVAC system. 
+ 
+**Function Call:**  None
+
+## Logic:    
+- Get the number of zone schedules in the list sent to the function: `num_zones = len(zone_OA_CFM_list_of_schedules)`  
+- For each hour of the year: `for x in Range(8760):`  
+    - Reset hourly_running_OA_CFM_total to zero: `hourly_zone_running_OA_CFM_total = 0`  
+    - For each zone schedule in zone_OA_CFM_list_of_schedules: `for i in Range(num_zones):`  
+        - Add zone's OA CFM hourly value to hourly_running_OA_CFM_total: `hourly_running_OA_CFM_total = hourly_running_OA_CFM_total + zone_OA_CFM_list_of_schedules[i][x]`      
+    - Add total hourly OA CFM to aggregated schedule for the hour: `aggregate_min_OA_schedule_across_zones[x] = hourly_running_OA_CFM_total`  
+
```

Comment:
@claperle if I understand correctly, we're creating an array of 8760 values which shows the min CFM across all zones being served by a HVAC system (as suggested by line 3), correct?
The term aggregate_min_OA_schedule_across_zones suggests the aggregate is a fractional value. Could we simply rename this aggregate_min_OA_CFM_schedule_across_zones

Reply:
Sure, do I just rename it in the RDS?